getting-started/hardhat/hardhat.config.js

```diff
@@ -19,9 +19,9 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
 module.exports = {
   solidity: "0.8.4",
   networks: {
-    optimistic: {
-      url: 'http://127.0.0.1:8545',
-      accounts: { mnemonic: 'test test test test test test test test test test test junk' }
+    "optimistic-kovan": {
+       url: "https://kovan.optimism.io",
+       accounts: { mnemonic: "test test test test test test test test test test test junk" }
     }
```

Comment:
Can we please add the network config for local and mainnet? I'd like if these tuts can be used as the base of a real project. As much as we can help them not have to think about things, the better.